README.md

```diff
@@ -155,6 +156,11 @@
 - [Thomas George Thomas](https://github.com/Thomas-George-T/Thomas-George-T)
 - [Derek Nguyen](https://github.com/dereknguyen269/dereknguyen269)
 
+#### Retro 😎
+
+- [Livio Brunner](https://github.com/BrunnerLivio)
+- [Christian Petersen](https://github.com/fnky)
```

Comment:
change links to `https://github.com/BrunnerLivio/BrunnerLivio` from `https://github.com/BrunnerLivio`

Reply:
Fixed